### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by Loic-Code**

### CODE REVIEW

**Review by ChristianOlmedo — DISMISSED** [review]

File: public/assets/css/footer.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        margin-top:20px;
+    }
+}
+
+@media (max-width:767px) {
+    .footer-dark .item.text {
+        margin-bottom:0;
+    }
+}
+
```
Comment (LucyMerlier) [inline public/assets/css/footer.css]: magic values + spaces after colons

File: src/View/Includes/_footer.html.twig [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                        <li><a href="#">Cours</a></li>
+                        <li><a href="#">Stage</a></li>
+                    </ul>
+                </div>
+                <div class="col-md-4 col-sm-12 item text">
+                    <h3>Haras du Val de Loire</h3>
+                    <ul>
+                        <li>Adresse : 999 rue de Chemeau 45560 Saint Denis en Val</li>
+                        <li>N° de contact : 02 38 24 75 20</li>
+                        <li>Email : contact@harasduvaldeloire.fr</li>
```
Comment (LucyMerlier) [inline src/View/Includes/_footer.html.twig]: warning : writing phone numbers and email addresses flat on a web page makes it easy for bots to pick up the informations and add them to spam lists

File: src/View/Includes/_footer.html.twig [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<div class="footer-dark">
+    <footer>
+        <div class="container">
+            <div class="row text-center">
+                <div class="col-sm-6 col-md-4 item">
+                    <h3>A propos de nous</h3>
```
Comment (LucyMerlier) [inline src/View/Includes/_footer.html.twig]: honnest question : are there h2s on this page before these h3s?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/Includes/_footer.html.twig (hunk 3 above)
Comment (Loic-Code) [reply src/View/Includes/_footer.html.twig]: he may have a h2 later

File: src/View/Includes/_footer.html.twig (hunk 2 above)
Comment (Loic-Code) [reply src/View/Includes/_footer.html.twig]: how am i supposed to do ?

**Review by NEL45 — DISMISSED** [review]

File: public/assets/css/footer.css [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        text-align: center;
+        margin-top: 20px;
+    }
+}
+
+@media (max-width:767px) {
+    .footer-dark .item.text {
+        margin-bottom: var(--margin-bottom);
+    }
+}
```
Comment (LucyMerlier) [inline public/assets/css/footer.css]: magic values

File: src/View/Includes/_footer.html.twig [hunk 5]
```diff
@@ -0,0 +1,35 @@
+<div class="footer-dark">
```
Comment (LucyMerlier) [inline src/View/Includes/_footer.html.twig]: why this extra div around the footer?

File: src/View/Includes/_footer.html.twig [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                    <ul>
+                        <li><a href="#">Accueil</a></li>
+                        <li><a href="#">Nous contacter</a></li>
+                    </ul>
+                </div>
+                <div class="col-sm-6 col-md-4 item">
+                    <h2>Services</h2>
+                    <ul>
+                        <li><a href="#">Cours</a></li>
+                        <li><a href="#">Stage</a></li>
```
Comment (LucyMerlier) [inline src/View/Includes/_footer.html.twig]: "stage", singular?

File: src/View/Includes/_footer.html.twig [hunk 7]
```diff
@@ -0,0 +1,35 @@
+<div class="footer-dark">
+    <footer>
+        <div class="container">
```
Comment (LucyMerlier) [inline src/View/Includes/_footer.html.twig]: why this extra div in the footer? couldn't those classes be directly applied to the footer tag?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

**Review by ChristianOlmedo — DISMISSED** [review]

**Review by LucyMerlier — DISMISSED** [review]

**Review by ChristianOlmedo — DISMISSED** [review]

**Review by bouboumael — APPROVED** [review]

File: src/View/layout.html.twig [hunk 8]
```diff
@@ -4,14 +4,16 @@
...
-		{% endblock %}
+	{% block title %}
+		Haras du Val de Loire
+	{% endblock %} 
 	</title>
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
-	<link rel="stylesheet" href="/assets/css/style.css"> {% block stylesheet %}{% endblock %}
+	<link rel="stylesheet" href="/assets/css/style.css">
+    {% block stylesheet %}{% endblock %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/ionicons/2.0.1/css/ionicons.min.css">
```
Comment (LucyMerlier) [inline src/View/layout.html.twig:16]: indentation?

File: src/View/layout.html.twig [hunk 9]
```diff
@@ -23,6 +25,9 @@
 	{% block button %}
 		{% include "Includes/_button_to_go_up.html.twig" %}
 	{% endblock %}
+    {% block footer %}
+		{% include 'Includes/_footer.html.twig' %}
+    {% endblock %}
```
Comment (LucyMerlier) [inline src/View/layout.html.twig:30]: indentation?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
indentation, see comments

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**